modules/dxp/apps/oauth/oauth-web/src/main/java/com/liferay/oauth/web/internal/struts/OAuthAccessTokenAction.java

```diff
@@ -12,7 +12,7 @@
  *
  */
 
-package com.liferay.oauth.web.internal.action;
+package com.liferay.oauth.web.internal.struts;
```

Comment:
@dantewang @yuhai ,
You forgot `BaseSamlStrutsAction` and its sub classes, they are not following package name rule.

Tina.